mitodl/open-discussions
# Prevent non-superusers from editing a channel title

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1537 

#### What's this PR do?
Prevents non-superusers from editing a channel title. Non-superusers will see a disabled title field in the channel appearance settings page, and any request to the API to change the title with a non-superuser will be denied

#### How should this be manually tested?
- Log in as a superuser and a non-superuser. Confirm that the channel title field is enabled and disable (respectively) on the channel appearance settings page
- Try to send a patch request to update a channel in a django shell using the django client. If the client is logged in with a non-superuser and the request data includes a `title` key, it should result in a 403

#### Screenshots (if appropriate)
![ss 2018-12-06 at 12 27 48](https://user-images.githubusercontent.com/14932219/49601066-cefb2480-f952-11e8-9c8f-d3bd1c9989e7.png)
